blocks/init/src/Blocks/custom/table-of-contents/table-of-contents-style.scss

```
@@ -0,0 +1,87 @@
+.block-table-of-contents {
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+
+	position: sticky;
+	top: 2rem;
+
+	height: fit-content;
+
+	&__entry {
+		@extend %button-reset;
+
+		background: none;
+
+		display: flex;
+		align-items: center;
+
+		transition: {
+			property: color;
+			timing-function: ease-out;
+			duration: 0.3s;
+		}
+
+		&:hover {
+			color: var(--global-colors-primary500);
+		}
+
+		&[data-level='2'],
+		&[data-level='3'],
+		&[data-level='4'],
+		&[data-level='5'],
+		&[data-level='6'] {
```

```suggestion
		&:not([data-level='1']) {
```
@goranalkovic-infinum Would this work?

---

Ah, yes, makes sense now 😄